spec/reek/cli/options_spec.rb

```diff
@@ -2,23 +2,25 @@
 require_lib 'reek/cli/options'
 
 RSpec.describe Reek::CLI::Options do
+  let(:options) { described_class.new }
```

Comment:
I disagree 100% that this is better than using `subject`. 
@waldyr what's your take on this?